src/codeu/chat/database/Database.java

```diff
@@ -0,0 +1,91 @@
+package codeu.chat.database;
+
+import codeu.chat.common.Conversation;
+import codeu.chat.common.Message;
+import codeu.chat.common.User;
+import codeu.chat.util.Uuid;
+import com.mongodb.MongoClientURI;
+import org.bson.Document;
+
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+public class Database {
+    public MongoClient mongoClient;
+    public MongoDatabase database;
+    public MongoCollection<Document> users;
+    public MongoCollection<Document> messages;
+    public MongoCollection<Document> conversations;
+
+    private boolean clearDatabase = false;
+    private String username = "User";
+    private String password = "SuperSecretPassword";
+    private String databasePath = "TestDatabase";
+    private String URI = "mongodb://" + username + ":" + password + "@codeu-shard-00-00-kskbs.mongodb.net:27017,codeu-shard-00-01-kskbs.mongodb.net:27017,codeu-shard-00-02-kskbs.mongodb.net:27017/" + databasePath + "?ssl=true&replicaSet=CodeU-shard-0&authSource=admin";
+
+    public Database(){
+        try{
+            MongoClientURI uriConnect = new MongoClientURI(URI);
+            this.mongoClient = new MongoClient(uriConnect);
+            this.database = mongoClient.getDatabase(databasePath);
+            System.out.println("Successful connection to database.");
+
+            if(clearDatabase){
+                removeDatabaseCollections();
+            }
+
+            createDatabaseCollections();
+
+        }catch(Exception e){
+            System.err.println( e.getClass().getName() + ": " + e.getMessage() );
```

Comment:
Still not _quite_ sure what you mean. I did go ahead and change the types of exceptions it captures, though. For example, now a failure to connect to the database results in a MongoExcception being throw (it extends RuntimeException).